assert/assertions_test.go

```diff
@@ -1644,49 +1644,33 @@ func TestLen(t *testing.T) {
 	ch <- 3
 
 	cases := []struct {
-		v interface{}
-		l int
+		v      interface{}
+		l      int
+		format string
```

Comment:
Rename to `expected1234567` (because this isn't a format but the expected message) and add a comment `// message when expecting 1234567 items`.

Reply:
Done